xwiki-platform-core/xwiki-platform-panels/xwiki-platform-panels-ui/src/main/resources/Panels/PanelSheet.xml

```diff
@@ -53,14 +53,21 @@
 #set ($obj = $doc.getObject('Panels.PanelClass'))
 #if ($obj)
   #set($class = $obj.xWikiClass)
-  #displayPanelProperty($obj 'name')
-  #displayPanelProperty($obj 'type')
-  #displayPanelProperty($obj 'category')
-  #displayPanelProperty($obj 'description')
-  #displayPanelProperty($obj 'content')
-  #displayPanelProperty($obj 'async_enabled')
-  #displayPanelProperty($obj 'async_cached')
-  #displayPanelProperty($obj 'async_context')
+  ## Enclose in .xform class div when editing
+  #if ($xcontext.action == 'edit')
+    {{html clean=false}}&lt;div class='xform'&gt;{{/html}}
+  #end
+    #displayPanelProperty($obj 'name')
```

Comment:
The issue I have is that it's not sufficient to have the .xform class added. 

If you read the .xform specification at https://www.xwiki.org/xwiki/bin/view/Documentation/DevGuide/FrontendResources/VerticalForms/ you will see that `dt` elements also have an additional `label` tag. This results in the form labels to be displayed in uppercase. You still need to do these changes.

Reply:
I agree. The structure is not complete without proper labels. I will implement the missing label tags asap.